src/Orchard.Web/Core/Navigation/Migrations.cs

```diff
@@ -177,5 +181,13 @@ public class Migrations : DataMigrationImpl {
 
             return 6;
```

Comment:
Right, the diff confused me. But then move all new WithIdentity you added to the `UpdateFrom6` as new installs will always go there.